packages/kitchen/src/components/Spacer/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styled from "styled-components";
+import { KitchenComponent } from "../../types";
+import { Gap } from "../../types/theme";
+
+export type SpacerProps = KitchenComponent & {
+  x?: number | keyof Gap;
+  y?: number | keyof Gap;
+};
+
+const Spacer = styled(({ ...props }: SpacerProps) => {
+  return <span {...props}></span>;
+})<SpacerProps>`
+  display: block;
+  width: 1px;
+  height: 1px;
+  min-width: 1px;
+  min-height: 1px;
+  // horizontal margin can be set with x and it can either be a number or a key from the theme
+  margin-left: ${(props) =>
```

Comment:
to write less you could destructure it

Reply:
what do you mean?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment